blackjack/src/main/kotlin/blackjack/view/PlayCommand.kt

```diff
@@ -0,0 +1,26 @@
+package blackjack.view
+
+enum class PlayCommand(
+    val value: String,
+) {
+
+    YES("y"),
+    NO("n")
+    ;
+
+    fun isYes(): Boolean {
+        return this == YES
+    }
+
+    fun isNo(): Boolean {
+        return this == NO
+    }
+
+    companion object {
+        fun of(value: String): PlayCommand {
+            return PlayCommand.values()
+                .find { it.value == value }
+                ?: throw IllegalArgumentException("존재하지 않는 명령어 입니다.")
+        }
```

Comment:
```suggestion
        fun of(value: String): PlayCommand {
            return PlayCommand.values().find { it.value == value }
                ?: throw IllegalArgumentException("존재하지 않는 명령어 입니다.")
        }
```

소소하지만, 이렇게 개행하면 정상케이스와 예외 케이스를 명확히 볼 수 있을 것 같아요.